dynamiqs/time_tensor.py

```diff
@@ -378,3 +424,91 @@ def __add__(self, other: Tensor | TimeTensor) -> TimeTensor:
             return PWCTimeTensor(factors, tensors, static=static)
         else:
             return NotImplemented
+
+
+class _ModulatedFactor:
+    # Defined by 2 tensors (factor, factor0), where
+    # - factor: (...,) is a callable that takes a time and returns a tensor
+    # - factor0: (...,) is the tensor returned by factor(0.0)
+    # factor0 holds information about the shape of the tensor returned by factor(t).
+
+    def __init__(self, factor: callable[[float], Tensor], factor0: Tensor):
+        self.factor = factor  # (...)
+        self.factor0 = factor0  # (...)
+
+    @property
+    def shape(self) -> torch.Size:
+        return self.factor0.shape
+
+    def conj(self) -> _ModulatedFactor:
+        return _ModulatedFactor(lambda t: self.factor(t).conj(), self.factor0.conj())
```

Comment:
Can we split like done in `CallableTimeTensor`? I think it's easier to read
```suggestion
        f = lambda t: self.f(t).conj()
        f0 = self.f0.conj()
        return _ModulatedFactor(f, f0)
```
Also in `view`.